components/PostArticle.vue

```diff
@@ -1,62 +1,52 @@
 <template>
-	<article>
-		<h1 v-html="title" class="title" />
-		<img v-if="picture" :src="picture" class="news-item-img">
-		<WeCountLogo v-else class="news-item-img" />
-		<div v-html="content" class="api-content" />
-		<div v-for="(t, index) in tags" :key="index">
-			<nuxt-link :to="{ path: '/tag', query: { s: t }}">
-				{{ t }}
-			</nuxt-link>
-		</div>
+	<article class="post-article">
+		<h1 v-html="postToRender.title" class="title" />
+
+		<div v-html="postToRender.author" class="author" />
+
+		<div v-html="postToRender.date" class="date" />
```

Comment:
The space between the date and content is larger in the Figma file than it appears here